src/Driver.ml

```diff
@@ -124,7 +128,7 @@ let detect_base_tools () =
     if read_one_line "uname" [||] = "Darwin" && !readlink = "readlink" then
       KPrint.bprintf "Warning: OSX detected and no greadlink found. Suggestion: \
         [brew install coreutils]\n";
-  with Process.Exit.Error _ ->
+  with _ ->
```

Comment:
same here, since you now have a catch-all in success, what kind of exception could possibly happen here?